app/src/components/navigation/TopBar.vue

```diff
@@ -1,17 +1,21 @@
 <script setup lang="ts">
 import { PlayCircleIcon, ChevronLeftIcon } from "@heroicons/vue/24/solid";
 import ProfileMenu from "./ProfileMenu.vue";
-import { useRoute, RouterLink } from "vue-router";
+import { useRoute } from "vue-router";
 import DesktopMenu from "./DesktopMenu.vue";
 
 const route = useRoute();
```

Comment:
```suggestion
const router = useRouter();
```

Reply:
useRoute is still needed for some of the css so that needs to be kept in, but I did instead use router instead of window.history.back() now, because useRouter() makes more sense here like you suggested.